crates/sncast/src/helpers/fee.rs

```diff
@@ -120,27 +188,22 @@ impl FeeArgs {
 #[derive(Debug, PartialEq, CairoDeserialize)]
 pub struct ScriptFeeSettings {
```

Comment:
At cast script level this should take an enum
```
pub enum ScriptFeeSettings {
  MaxFee(value),
  GasTriplet(...)
}
```

That's a feature for future versions but please create an issue for that.